core/modules/field/field.block.inc

```diff
@@ -32,11 +32,13 @@ class FieldBlock extends Block {
   function getAdminPreview() {
     list($entity_type, $field_name) = explode('-', $this->childDelta, 2);
     $field = field_info_field($field_name);
+    $field_cardinality = $field['cardinality'];
     $formatter = field_info_formatter_types($this->settings['formatter']);
     $pseudo_instance = $this->createPseudoInstance($this->settings['formatter'], $this->settings['formatter_settings']);
     $summary = module_invoke($formatter['module'], 'field_formatter_settings_summary', $field, $pseudo_instance, '_custom');
 
-    $preview = '<p>' . t('Displays values of "%field" field.', array('%field' => $field_name)) . '</p>';
+    $preview = format_plural($field_cardinality, 'Displays the value of the %field field.', 'Displays values of the %field field.', array('%field' => $field_name));
```

Comment:
This is a great improvement, but while we are changing translated strings I'd also like to make this more accurate :) This block does not only display the value of the field (example `foo`), it displays the complete field, both formatted and rendered (example: `<div class="field field-body><p>foo</p></div>`).

Let's remove "Values of the" and make the description more truthful :)

```
$preview = t('Displays the %field field.', array('%field' => $field_name));
```